genie-web/src/test/java/com/netflix/genie/web/health/GenieHealthIndicatorUnitTests.java

```diff
@@ -41,7 +41,7 @@
     private static final int DEFAULT_JOB_MEMORY = 1_024;
     private static final int MAX_JOB_MEMORY = 5_120;
 
-    private GenieHealthIndicator genieHealthIndicator;
```

Comment:
You should rename this test class file if you're also renaming the class it's testing